bots/images/scripts/fedora.setup

```diff
@@ -60,6 +62,12 @@ tuned \
 virt-install \
 "
 
+# HACK docker not available on f31
+# https://github.com/cockpit-project/cockpit/issues/12670
+if [ "$IMAGE" != fedora-31 ]; then
+    COCKPIT_DEPS="$COCKPIT_DEPS atomic docker"
+fi
```

Comment:
If that's a temporary hack, fine. If it's planned (or at least likely) that docker just won't be a thing any more in F31, then let's swap this around: Only add docker to fedora-30 (legacy mode).